docs/reference/skin.md

```diff
@@ -0,0 +1,290 @@
+## Skin
+
+Derived from [Device](device.md).
+
+```
+Skin {
+  SFVec3f      translation     0 0 0
+  SFRotation   rotation        0 1 0 0
+  SFVec3f      scale           1 1 1
+  SFString     name            "skin"
+  SFString     modelUrl        ""
+  MFNode       appearance      []
+  MFNode       bones           []
+  SFBool       castShadows     TRUE
+  SFFloat      translationStep 0.01
+  SFFloat      rotationStep    0.261799387
+}
+```
+
+### Description
+
+The [Skin](#skin) node can be used to simulate soft mesh animation for example of a human or an animal.
+The skin mesh is imported from a mesh file specified by the `model` name but in order to be animated it has to be attached to a skeleton so that the rotation of the skeleton joints results in appropriate deformation of the skin mesh.
```

Comment:
```suggestion
The skin mesh is imported from a mesh file specified by the `modelUrl` name but in order to be animated it has to be attached to a skeleton so that the rotation of the skeleton joints results in appropriate deformation of the skin mesh.
```